packages/storybook-html/src/components/card/card.docs.mdx

```diff
@@ -0,0 +1,116 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+import StatusBadge from '../../../.storybook/custom/components/StatusBadge';
+
+# Card
+
+<StatusBadge.Stable />
+
+<a href="https://github.com/uq-its-ss/design-system/issues" class="uq-button uq-button--text" target="_blank">Check for open issues</a>
+
+## Overview
+
+The card component is used to provide a brief summary of content or a task, often with a link to more detail. Cards are frequently displayed alongside other cards to group related content or tasks.
+
+## Default example
+
+### a. Card default
+
+<Canvas withSource="none">
+  <Story id="components-cards--card-default" />
+</Canvas>
+
+```html
+
+```
+
+### b. Card default with link
+
+<Canvas withSource="none">
+  <Story id="components-cards--card-default-with-link" />
+</Canvas>
+
+```html
+
+```
+
+## Variant examples
+
+### a. Card square overlay title
```

Comment:
I think you need to adjust the naming... these are 

Overlay cards 

Overlay card, Square

Reply:
Done